banners/mobile/content/messages/DynamicCampaignTextOverwrite.de.ts

```diff
@@ -1,19 +1,7 @@
 import { TranslationMessages } from '@src/Translator';
 
 const Translations: TranslationMessages = {
-
-	'date-month-time-1': '{{day}} Januar, um {{time}} Uhr',
-	'date-month-time-2': '{{day}} Februar, um {{time}} Uhr',
-	'date-month-time-3': '{{day}} März, um {{time}} Uhr',
-	'date-month-time-4': '{{day}} April, um {{time}} Uhr',
-	'date-month-time-5': '{{day}} Mai, um {{time}} Uhr',
-	'date-month-time-6': '{{day}} Juni, um {{time}} Uhr',
-	'date-month-time-7': '{{day}} Juli, um {{time}} Uhr',
-	'date-month-time-8': '{{day}} August, um {{time}} Uhr',
-	'date-month-time-9': '{{day}} September, um {{time}} Uhr',
-	'date-month-time-10': '{{day}} Oktober, um {{time}} Uhr',
-	'date-month-time-11': '{{day}} November, um {{time}} Uhr',
-	'date-month-time-12': '{{day}} Dezember, um {{time}} Uhr'
+	'campaign-day-only-n-days': 'Heute sind es nur noch {{days}} Tage bis zum Ende unserer Spendenkampagne.'
```

Comment:
You shouldn't do it like this as the campaign days sentence already has set up for it, instead you should set the `urgencyMessageDaysLeft` to a higher number than there are days left in this campaign.

See https://github.com/wmde/fundraising-banners/blob/f6d0eb86c336971b995cabbc14aac650d09a9c9f/banners/desktop/banner_var.ts#L63 for an example